### PR TITLE
DOC: Pin nipype with new sphinx extension syntax

### PR DESCRIPTION
**Pull request by effigies**
Testing nipy/nipype#3206.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -52,7 +52,7 @@ doc =
     packaging
     pydot >= 1.2.3
     pydotplus
-    sphinx >= 1.5.3, < 3
+    sphinx >= 1.5.3
```
Comment (oesteban) [inline setup.cfg:55]: when the new API of sphinx started? as early as 1.5.3?

Comment (effigies) [reply setup.cfg:55]: 1.8

Comment (oesteban) [reply setup.cfg:55]: Then
```suggestion
    sphinx >= 1.8
```
I guess?

Comment (effigies) [reply setup.cfg:55]: Yup. I've got a few commits I haven't pushed yet. I was just using this to test for the moment. I would say let's make this a real PR when we depend on 1.5.0.

**Pull request merged**